canvas_modules/common-canvas/src/common-properties/components/moveable-table-rows/moveable-table-rows.jsx

```diff
@@ -166,17 +195,32 @@ class MoveableTableRows extends React.Component {
 		const selected = this.props.controller.getSelectedRows(this.props.propertyId).sort(function(a, b) {
 			return a - b;
 		});
+		let leastValue = 0;
+		const staticRows = this.props.controller.getStaticRows(this.props.propertyId).sort();
+		if (staticRows && staticRows.length > 0 && staticRows.includes(0)) {
+			leastValue = staticRows[staticRows.length - 1];
+		}
```

Comment:
I see this piece of code is the same as the one added in `topMoveRow()`. Would this make sense to be a separate function?